perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -775,6 +773,20 @@ def Prepare(self, vm):
                                                      DEFAULT_GCS_REGION,
                                                      self.regional_bucket_name))
 
+    # Detect if we need to install crcmod for gcp.
+    # See "gsutil help crc" for details.
+    raw_result, _ = vm.RemoteCommand('%s version -l' % vm.gsutil_path)
+    logging.info('gsutil version -l raw result is %s', raw_result)
+    search_string = 'compiled crcmod: True'
+    result_string = re.findall(search_string, raw_result)
+    if len(result_string) == 0:
+      logging.info('compiled crcmod is not available, installing now...')
+      vm.RemoteCommand('sudo pip install -U crcmod')
```

Comment:
Done. Added the logic to uninstall first and then reinstall per documentation.

As to specific version, at this point I believe we should always get the latest version - I'm not worried about using the newest version of the crcmod, it's very stable (much more stable than Azure's python package which just reached 1.0). If it has a breaking change it'd break all gsutil users...